samples/image-classifier-augmented-manifest.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: sagemaker.aws.amazon.com/v1
```

Comment:
Can we link here more details on this example ?
For instance you can add tutorial, blog or any other useful doc as comment for reference ?

Reply:
I don't have one, sadly. I'm not sure where this example comes from. I did some research online but couldn't nail down any sources that matched.